en/components/grid_toolbar.md

```diff
@@ -43,12 +45,14 @@ The available export options are hosted in a drop down which is shown when the m
 <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="grid-toolbar-sample-2-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
 </div>
 
-The following code snippet demonstrates how to enable the MS Excel export and CSV export features on the toolbar and also set titles for the exporting buttons:
+The following code snippet demonstrates how to enable all features of the of the toolbar:
```

Comment:
'of the' twice ->enable all features **of the of the** toolbar:

Reply:
Done